### DISCUSSION TITLE
配置ssh nginx代理后，一直会存在端口。这个怎么去掉

### DISCUSSION THREAD

**Discussion by Sleeeeeeeping** [gogs/gogs#7636] [中文讨论]
<img width="508" alt="image" src="https://github.com/gogs/gogs/assets/108887396/3551598c-888a-44ed-a1d5-f66e914b037c">

**Comment by aohanhongzhi** [comment]
我也表示疑惑。

nginx反代支持git协议吗？ git:// 使用会不会还是有端口？

https://blog.mynook.info/post/host-your-own-git-server-using-gogs/
